rotkehlchen/chain/evm/decoding/weth/decoder.py

```diff
@@ -47,10 +50,15 @@ def __init__(
         self.counterparty = counterparty
 
     def _decode_wrapper(self, context: DecoderContext) -> DecodingOutput:
-        if context.tx_log.topics[0] == WETH_DEPOSIT_TOPIC:
+        input_data = context.transaction.input_data
+        if context.tx_log.topics[0] == WETH_DEPOSIT_TOPIC or input_data.startswith(
+            WETH_ARBITRUM_DEPOSIT_METHOD,
+        ):
```

Comment:
change the check to 

```suggestion
        if (
             context.tx_log.topics[0] == WETH_DEPOSIT_TOPIC or
             (
                  self.evm_inqurirer.chain_id == ChainID.ARBITRUM_ONE and
                  input_data.startswith(WETH_ARBITRUM_DEPOSIT_METHOD
             )
       ):
```

Reply:
Also add a comment explaining that weth in arbitrum is a proxy and you can only check the method called